docs/apache-airflow/concepts/dags.rst

```diff
@@ -564,7 +564,8 @@ doc_md      markdown
 doc_rst     reStructuredText
 ==========  ================
 
-Please note that for DAGs, ``doc_md`` is the only attribute interpreted.
+Please note that for DAGs, ``doc_md`` is the only attribute interpreted. For DAGs it can contain a string or the reference to a template file. Template references are recognized by str ending in ‘.md’.
```

Comment:
Also the file must exist.

Reply:
Add in the documentation, mentioning also the exception